src/components/common/FacebookButton.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import FacebookLogin from 'react-facebook-login';
+import { face_app_key } from '../../constants/constants';
+import { facebookLogin } from '../../actions/sessionActions';
+
+class FacebookButton extends Component {
```

Comment:
I don't see this component using any of the `Component` class lifecycle methods neither a local state, so I think it would be a good idea to write it down as a presentational component only.